tests/15-rpl-classic/js/12-rpl-rank-error.js

```diff
@@ -0,0 +1,15 @@
+TIMEOUT(600000, log.testFailed());
+
+while(true) {
+  YIELD();
+
+  if(msg.contains("RPL Option Error: Dropping Packet")) {
+    log.log(time + " " + "node-" + id + " "+ msg + "\n");
+    log.testFailed();
+  }
+
+  else if(msg.contains("Received all packets")) {
```

Comment:
Previous block returns, so the `else` can be removed.

Reply:
Fixed in #2548